Project-1/README.md

```diff
@@ -13,111 +13,345 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
 1. Provide the truth tables for each of the following propositional logic formulas. State whether each is a tautology, a contradiction, or contingent:
   ```(a) (¬A→B)∨((A∧¬C)→B) 
   A	B	C	((¬A → B) ∨ ((A ∧ ¬C) → B))
-F	F	F	T
-F	F	T	T
-F	T	F	T
-F	T	T	T
-T	F	F	T
-T	F	T	T
-T	T	F	T
-T	T	T	T
-
-(a) is a tautology.
+F	F	F		  T
+F	F	T		  T
+F	T	F		  T
+F	T	T		  T
+T	F	F		  T
+T	F	T		  T
+T	T	F		  T
+T	T	T		  T
+
+It is a Tautology.
    
   (b) (A→B)∧(A→¬B)
   
   A	B	((A → B) ∧ (A → ¬B))
-F	F	T
-F	T	T
-T	F	F
-T	T	F
-  (b) is contigent. 
+F	F		 T
+F	T		 T
+T	F		 F
+T	T		 F
+It is Contigent. 
   
   
   (c) (A→(B∨C))∨(C→¬A) 
   
   A	B	C	((A → (B ∨ C)) ∨ (C → ¬A))
-F	F	F	T
-F	F	T	T
-F	T	F	T
-F	T	T	T
-T	F	F	T
-T	F	T	T
-T	T	F	T
-T	T	T	T
- (c) is a tautology.
+F	F	F		       T
+F	F	T		       T
+F	T	F		       T
+F	T	T		       T
+T	F	F		       T
+T	F	T		       T
+T	T	F		       T
+T	T	T		       T
+It is a Tautology.
   
   (d) ((A→B)∧C)∨(A∧D) 
   
   A	B	C	D	(((A → B) ∧ C) ∨ (A ∧ D))
-F	F	F	F	F
-F	F	F	T	F
-F	F	T	F	T
-F	F	T	T	T
-F	T	F	F	F
-F	T	F	T	F
-F	T	T	F	T
-F	T	T	T	T
-T	F	F	F	F
-T	F	F	T	T
-T	F	T	F	F
-T	F	T	T	T
-T	T	F	F	F
-T	T	F	T	T
-T	T	T	F	T
-T	T	T	T	T
-(d) is contignet.
+F	F	F	F		       F
+F	F	F	T		       F
+F	F	T	F		       T
+F	F	T	T		       T
+F	T	F	F		       F
+F	T	F	T		       F
+F	T	T	F		       T
+F	T	T	T		       T
+T	F	F	F		       F
+T	F	F	T		       T
+T	F	T	F		       F
+T	F	T	T		       T
+T	T	F	F		       F
+T	T	F	T		       T
+T	T	T	F		       T
+T	T	T	T		       T
+It is Contignet.
+```
 	
 2. A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
-  ```(a) (A→B)→C
+  ```
+  (a) (A→B)→C
   
-  (A ∧ ¬B) ∨ C
+  (A ∨ C) ∧ (¬B ∨ C)
   
   (b) (A→(B∨C))∨(C→¬A)
   
-  
+  This is a tautology, so no CNF form is possible. p∨~p
   
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
+  
+  This is a tautology, so no CNF form is possible. p∨~p
 ```
 
 3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
-  ```(a)  B(x,y) that says that x is a brother of y
+  ```
+  (a)  B(x,y) that says that x is a brother of y
```

Comment:
The variables in B(x,y) should be under the scope of universal quantifiers for a definition. What you've written below is a definition for 'brother of' that is satisfied by any one x,y pair, rather than conditions for any pair to satisfy 'brother of'